File: app/src/main/java/com/lipeng/mygithub/util/ListUtils.java
package com.lipeng.mygithub.util;

import java.util.List;

/**
 * List工具类
 * @author lipeng
 * @date 2018/3/16
 */

public final class ListUtils {

    private ListUtils(){

    }

    public static <T> int getSize(List<T> list){
        return null == list ? 0 : list.size();
    }

    public static <T> boolean isEmpty(List<T> list){
        return null == list || 0 == list.size() ;
    }
}

File: app/src/main/java/com/lipeng/mygithub/util/ListUtils.kt
package com.lipeng.mygithub.util

/**
 * @author big insect
 */
object ListUtils {
    fun <T> getSize(list: List<T>?):Int{
        return list?.size ?: 0
    }

    fun <T> isEmpty(list: List<T>?):Boolean{
        return list == null || list.isEmpty()
    }
}